src/neo/Network/P2P/Payloads/WitnessScope.cs

```diff
@@ -2,35 +2,38 @@
 
 namespace Neo.Network.P2P.Payloads
 {
+    /// <summary>
+    /// Represents the scope of a <see cref="Witness"/>.
+    /// </summary>
     [Flags]
     public enum WitnessScope : byte
     {
         /// <summary>
-        /// No contract was witnessed. Only sign the transaction.
+        /// Indicates that no contract was witnessed. Only sign the transaction.
```

Comment:
Somehow mention that this can be used to pay fees if signer is a sender (and not really useful otherwise)?